src/System.Private.ServiceModel/src/System/ServiceModel/Channels/MessageContent.cs

```diff
@@ -15,9 +15,11 @@ namespace System.ServiceModel.Channels
 {
     public abstract class MessageContent : HttpContent
     {
+#pragma warning disable 3008	// not CLS-compliant
         protected Message _message;
         protected MessageEncoder _messageEncoder;
         protected Stream _stream = null;
+#pragma warning restore 3008	// not CLS-compliant
```

Comment:
The compliance issue is that protected fields can't start with an underscore. What's the naming standard for protected fields? I would rather change the field names rather then disable the warning.